### ISSUE TITLE
Reproducing Results in Table 1

### ISSUE THREAD

**Issue by alasdairtran** [zongyi-li/fourier_neural_operator#30]
Hi Zongyi,

I really enjoyed reading your paper! I was having a go at reproducing your numbers in Table 1. I'm wondering if the numbers on the last row come from the `test_l2` variable of the final epoch?

If I clone your repo and run `python fourier_1d.py`, then I have `test_l2` as 0.00254360 on epoch 500. In Table 1 of your paper, you report a value of 0.0160 for the FNO model with s = 1024. This seems to be an order of magnitude bigger. Am I looking at the right metric?

**Comment by zongyi-li** [comment]
Hi Alasdair, in the paper, we used the burgers_R10.mat dataset with modes=16, width=64, and epoch=500. If you get better results probably you are using burgers_v1000.mat dataset, which has smoother initial conditions and therefore easier to model.

**Comment by alasdairtran** [comment]
No I was using `burgers_data_R10.mat` from your google drive. I didn't change any code in `fourier_1d.py`.

Upon closer inspection, I was able to get a `test_l2` closer to the number in Table 1 when I reverted back a few commits. I'm guessing that the one-order-of-magnitude improvement is due to the commit where you remove the batch norm. It's interesting that batch norm makes the performance a lot worse here.
